app/Http/Controllers/Twill/BaseController.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Http\Controllers\Twill;
+
+use A17\Twill\Models\Contracts\TwillModelContract;
+
+class BaseController extends ModuleController
+{
+    protected function setUpController(): void
+    {
+        $this->disablePermalink();
+        $this->enableSkipCreateModal();
+    }
+
+    public function edit(TwillModelContract|int $id): mixed
+    {
+        return parent::edit($id)->with([
+            'editableTitle' =>
+                $this->repository->isFillable($this->titleColumnKey)
+                || $this->repository->isTranslatable($this->titleColumnKey),
+        ]);
+    }
```

Comment:
This prevents Twill's special title field on the edit page from being editable if the model's title field isn't actually fillable.